lib/parse-all.js

```diff
@@ -26,9 +26,15 @@ function parseAll(config) {
 					return { name: name, error: err };
 				});
 		})
-	);
+	).then(result => {
+		const errors = result.filter(error => error);
```

Comment:
I'd change the signature of the [`.catch()` on line 28](https://github.com/voorhoede/fastatic/pull/97/files#diff-c5b236071f712aa9d2ab79e257a82b95R28) to `return { error: err }`:
```javascript
.catch(err => {
  err.parser = name;
  err.cwd = config.temp.dest;
  parsing[name].reject(err);
  return { error: err };
});
```
That way you can change the following as well:
```javascript
.then(results => {
  progressBar.stop();

  const errors = results
    .map(result => result.error)
    .filter(error => error !== undefined);

  if (errors.length) {
    throw errors;
  }
});
```